extensions/EntityExtension/src/main/kotlin/com/typewritermc/entity/entries/data/minecraft/living/armorstand/InvisibleData.kt

```diff
@@ -0,0 +1,38 @@
+package com.typewritermc.entity.entries.data.minecraft.living.armorstand
+
+import com.typewritermc.core.books.pages.Colors
+import com.typewritermc.core.extension.annotations.Entry
+import com.typewritermc.core.extension.annotations.Tags
+import com.typewritermc.engine.paper.entry.entity.SinglePropertyCollectorSupplier
+import com.typewritermc.engine.paper.entry.entries.EntityData
+import com.typewritermc.engine.paper.entry.entries.EntityProperty
+import com.typewritermc.engine.paper.extensions.packetevents.metas
+import me.tofaa.entitylib.meta.other.ArmorStandMeta
+import me.tofaa.entitylib.wrapper.WrapperEntity
+import org.bukkit.entity.Player
+import java.util.*
+import kotlin.reflect.KClass
+
+@Entry("invisible_data", "Whether the armor stand itself is invisible", Colors.RED, "mdi:mirror-variant")
+@Tags("invisible_data", "armor_stand_data")
+class InvisibleData(
+    override val id: String = "",
+    override val name: String = "",
+    val isInvisible: Boolean = false,
```

Comment:
To keep this consistent with other properties like `agable` and `glowing`, rename it to `invisible` instead of `isInvisible`.